mos_tests/rabbitmq_oslo/test_restarts.py

```diff
@@ -236,6 +235,32 @@ def consume_msg(remote, cfg_file_path):
     num_of_msg_consumed = int(re.findall('\d+', out_consume)[0])
     return num_of_msg_consumed
 
+
+def rabbit_rpc_server_start(remote, cfg_file_path):
+    logger.debug('Start [oslo_msg_check_server] on %s.' % remote.host)
+    cmd = 'oslo_msg_check_server --nodebug --config-file {0}'.format(
+        cfg_file_path)
+    remote.execute_async(cmd)
```

Comment:
This is because I need to keep server/client running on node and continue test-code execution.
`check_call` or `execute` are waiting for command to finish it's execution.
But in my case server/client is running and keep posting information in console.